### PR TITLE
Bump version to 28.3.0

### PR DESCRIPTION
**Pull request by DilwoarH**
Adds in text diff to frontend toolkit

Ticket: https://trello.com/c/a90iPGOH/117-move-ins-del-to-frontend-toolkit

### CODE REVIEW

File: VERSION.txt [hunk 1]
```diff
@@ -1 +1 @@
-28.1.0
+28.1.1
```
Comment (idavidmcdonald) [inline VERSION.txt]: Minor bump not patch as new functionality

**Comment by DilwoarH** [comment]
@idavidmcdonald - updated

**Comment by idavidmcdonald** [comment]
Commit message needs updating too please

**Comment by DilwoarH** [comment]
@idavidmcdonald - thanks updated

**Pull request merged**